mw-webapp/src/logic/allUsersPage/AllUsersPage.module.scss

```diff
@@ -15,11 +15,36 @@
   justify-content: space-between;
 }
 
-.inputFilter {
+.inputWrapper {
```

Comment:
Maybe we need to create a separate component for FilterInput? Because we use it on many pages) What do you think?

Reply:
Create FilterInput component with SearchIcon?